examples/cis-glb/README.md

```diff
@@ -1,21 +1,21 @@
-# Example cis-glb
+# Example cis_glb
 
 This example creates Domain and manages GLB and origin pools for an existing instance .
 
 ## Example Usage
 
 ```terraform
-module "cis-domain" {
+module "cis_domain" {
   source                = "../../modules/domain"
   is_cis_instance_exist = true
   service_name          = var.service_name
```

Comment:
Could you put an example of value instead of var. references - this makes easier for consumer to deduce what type of value of valid through example when ambiguity exists.